<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>531</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>Search</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Category</string>
     </property>
    </widget>
    <widget class="QComboBox" name="category">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>151</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>database</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>OOP</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>AI</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Title</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="title">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>60</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="issuedbox">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>120</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Issued</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>20</y>
       <width>131</width>
       <height>111</height>
      </rect>
     </property>
     <property name="title">
      <string>Type</string>
     </property>
     <widget class="QRadioButton" name="reference">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Reference book</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>95</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Journal</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="textbook">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>95</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Textbook</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="searchbutton">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>140</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
   </widget>
   <widget class="QTableWidget" name="booksTableWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>531</width>
      <height>192</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>1</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>2</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>3</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>4</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>5</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>6</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>ISBN</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Title</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Category</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Type</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Issued</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="viewbutton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>440</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>View</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deletebutton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>440</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="closebutton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>480</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
